runtest.sh

```diff
@@ -0,0 +1,9 @@
+RAILS_ENV=test rails test test/features/filtering_event_list.rb test/features/editing_event_review_test.rb test/integration/view_event_test.rb
```

Comment:
This is new. How does that work? Is this your local test code committed by mistake?

Reply:
Mistake